tokio-trace/src/span.rs

```diff
@@ -699,7 +699,7 @@ impl EnteredUnscoped {
     }
 
     /// Borrows the entered `Span`.
-    pub fn span(&self) -> &Span {
+    fn span(&self) -> &Span {
```

Comment:
I thought we shouldn't _commit_ to exposing this publically in this PR, since all of the inner functions are still exposed. There's no _obvious_ reason it would cause problems to, but I thought it was good to hedge our bets for now. We can look into making this `pub` if there's a use-case and we determine this won't introduce any issues.

Reply:
![Screen Shot 2019-06-18 at 1 02 24 PM](https://user-images.githubusercontent.com/2796466/59722261-19017480-91d8-11e9-9d0e-0400befd6efe.png)
☝️ is the objectively correct name